lxd/daemon.go

```diff
@@ -899,6 +899,7 @@ func (d *Daemon) Init() error {
 			caPool.AddCert(ca)
 			tlsConfig.RootCAs = caPool
 			tlsConfig.ClientCAs = caPool
+			tlsConfig.InsecureSkipVerify = false
```

Comment:
That is, we don't go through an initialization function for tls.Config, we instead define all the members of the struct, so any boolean will be false at that point.